src/averageAges.js

```diff
@@ -14,12 +14,25 @@
  *
  * @return {number}
  */
+
+function calculateSumAges(people) {
+  const sumAges = people.reduce((ages, person) =>
+    (person.died - person.born) + ages, 0);
+
+  return sumAges / people.length;
+}
+
+function getPersonsGender(people, gender) {
+  return people.filter(person => person.sex === gender);
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mens = century
+    ? getPersonsGender(people, 'm')
+      .filter(person => Math.ceil(person.died / 100) === century)
+    : getPersonsGender(people, 'm');
```

Comment:
better create separate variable with result of `getPersonsGender`, don't repeat yourself